lib/private/Notification/Manager.php

```diff
@@ -141,7 +150,33 @@ protected function getApps(): array {
 	 * @return INotifier[]
 	 */
 	public function getNotifiers(): array {
-		if (empty($this->notifierClasses)) {
+		if (!$this->parsedRegistrationContext) {
+			$notifierServices = $this->coordinator->getRegistrationContext()->getNotifierServices();
+			foreach ($notifierServices as $notifierService) {
+				try {
+					$notifier = \OC::$server->query($notifierService->getService());
+				} catch (QueryException $e) {
+					$this->logger->logException($e, [
+						'message' => 'Failed to load notification notifier class: ' . $notifierService->getService(),
+						'app' => 'notifications',
+					]);
+					continue;
+				}
+
+				if (!($notifier instanceof INotifier)) {
+					$this->logger->error('Notification notifier class ' . $notifierService->getService() . ' is not implementing ' . INotifier::class, [
+						'app' => 'notifications',
+					]);
+					continue;
+				}
+
+				$this->notifiers[] = $notifier;
+			}
+
+			$this->parsedRegistrationContext = true;
+		}
+
+		if (!empty($this->notifiers)) {
```

Comment:
```suggestion
		if (empty($this->notifierClasses)) {
```

And we are good to go

Reply:
Yes. That change was still pending